java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3023,4 +3024,69 @@ public void touch(BaseClient client) {
         assertEquals(2, client.touch(new String[] {key1, key2}).get());
         assertEquals(0, client.touch(new String[] {key3}).get());
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void bitcount(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        String value = "foobar";
+
+        assertEquals(OK, client.set(key, value).get());
+        assertEquals(26, client.bitcount(key).get());
+    }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void bitcount_with_indices(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        String value = "foobar";
+
+        assertEquals(OK, client.set(key, value).get());
+        assertEquals(6, client.bitcount(key, 1, 1).get());
+    }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void bitcount_with_BYTE_indices(BaseClient client) {
```

Comment:
bit/byte feature added in redis 7
Use `assume` to check redis version in tests

Reply:
Thanks! Do we only check for redis version after 7.0.0? What about 6.0.0 and others?

Reply:
We check it for commands were added/modified in other versions.

Reply:
Right but how come I don't see other `assume`s for version < 7.0.0. Can we assume that testing has a base version of 6.0.0?

Reply:
Yes, we officially support versions 6+ (see main readme).
Ci runs on 6 and 7